layouts/blog/baseof-hi.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html id="blog" lang="{{ .Language }}" class="gr__kubernetes_io fontawesome-i2svg-active fontawesome-i2svg-complete">
+	<head>
+		{{ partial "head.html" . }}
+		<link rel='stylesheet' type='text/css' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css'>
+		<script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+		<link rel="stylesheet" type="text/css" href="{{ "css/blog.css" | relURL }}">
+	</head>
+	<body>
+		{{ partial "header.html" . }}
+		<section id="hero" class="light-text no-sub">
+			<h1>{{ .Title }}</h1>
+		</section>
+		<div class="container-fluid" style="padding-top: 2em">
+			<div class="row">
+				<div class="col-xs-12 col-md-9 text blog-content">
+					{{ block "main" . }}{{ end }}
+				</div>
+				<div class="col-xs-12 col-md-3 text">
+					<div class="widget-content">
+						{{ with site.Home.OutputFormats.Get "rss" -}}
+						<link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}" title="{{ site.Title }}">
+						<a class="widget-link" href="{{ .Permalink | safeURL }}"> <i class="fas fa-rss fab-icon"> </i> RSS Feed </a>
+						{{ end -}}
+						<a class="widget-link" href="https://kubernetes.io/docs/contribute/start/#write-a-blog-post"><i class="fa fa-edit fab-icon"></i> पोस्ट जमा करें</a>
+						<a class="widget-link" href="https://twitter.com/kubernetesio"> <i class="fab fa-twitter-square fab-icon"> </i> @Kubernetesio</a>
+						<a class="widget-link" href="https://github.com/kubernetes/kubernetes"> <i class="fab fa-github-square fab-icon"></i> Github पर देखें </a>
+						<a class="widget-link" href="http://slack.k8s.io"><i class="fab fa-slack fab-icon"> </i> #कुबेरनेट्स-यूज़र्स </a>
```

Comment:
Do not localize the name of slack channels, there is no channel name `#कुबेरनेट्स-यूज़र्स`

Reply:
Okay, sure!